cypress/integration/pages/articles.js

```diff
@@ -270,10 +270,19 @@ const runCanonicalTests = ({ service }) =>
 
     describe('ATI', () => {
       it('should have a noscript tag with an 1px image with the ati url', () => {
-        cy.hasNoscriptImgAtiUrl(
-          envConfig.atiUrl,
-          config[service].isWorldService ? envConfig.atiAnalyticsWSBucket : '',
-        );
+        if (!service.japanese) {
```

Comment:
What is this doing? is this doing `'japanese'.japanese`?

Reply:
Should it not be `service !== 'japanese'`?

Reply:
That's what I tried originally but Japanese is an object property name, it's not a string so that's why I used `!service.japanese`

Reply:
Also just seen that I forgot to add the logic to the liveRadio e2e's so I'm just about to do that

Reply:
But service comes from `const runCanonicalTests = ({ service }) =>` which comes from https://github.com/bbc/simorgh/blob/3bcd3aaefe1e0fbf60d0333c9271963ca8656cb2/cypress/support/iterator.js#L22 which comes from `Object.keys(config)` which is an array of strings 🤔

Reply:
![image](https://user-images.githubusercontent.com/11341355/63527027-46c8a980-c4f8-11e9-872c-e72acffd3cd9.png)